Game/tr/Game.po

```diff
@@ -1187,63 +1187,63 @@ msgstr "Depar"
 #: /Game/Resources/Localisation/ST_CaptureGuard01.ST_CaptureGuard01
 msgctxt "ST_CaptureGuard01,CaptureGuard01a"
 msgid "We thought we'd just let you keep wandering around the island, killing Sokol-Glockner personnel?"
-msgstr ""
+msgstr "Biz senin adada dolaşmana izin verdik, ama Sokol-Glockner çalışanını öldürmek?"
 
 #. Key:	CaptureGuard01b
 #. SourceLocation:	/Game/Resources/Localisation/ST_CaptureGuard01.ST_CaptureGuard01
 #: /Game/Resources/Localisation/ST_CaptureGuard01.ST_CaptureGuard01
 msgctxt "ST_CaptureGuard01,CaptureGuard01b"
 msgid "You're a disgrace. You think you know what's going on here? You think it's something YOU can stop?"
-msgstr ""
+msgstr "Sen rezaletsin. Burada ne olduğunun farkındamısın? Bunu SENİN durdurabileceğinimi sanıyorsun?"
```

Comment:
This could be google translate that's wrong, but it splits "farkındamısın" to "farkında mısın" and "durdurabileceğinimi" to "durdurabileceğini mi" - is it common to keep these together?